test/lib/jdk/test/lib/jvmti/jvmti_common.hpp

```diff
@@ -324,6 +324,18 @@ get_method_name(jvmtiEnv *jvmti, JNIEnv* jni, jmethodID method) {
   return mname;
 }
 
+static char*
+get_field_name(jvmtiEnv *jvmti, JNIEnv* jni, jclass field_class, jfieldID field) {
+  char* name = nullptr;
+  char* signature = nullptr;
+  char* generic = nullptr;
+  jvmtiError err = jvmti->GetFieldName(field_class, field, &name, &signature, &generic);
+  check_jvmti_status(jni, err, "get_field_name: error in JVMTI GetFieldName call");
+  deallocate(jvmti,jni, signature);
+  deallocate(jvmti,jni, generic);
```

Comment:
Nit: The lines 334 and 335 miss space after comma in the argument lists. Also, it is better to pass `nullptr` instead of `&generic`, so there would not be need to have this local and to deallocate the returned string.